src/types/Error.ts

```diff
@@ -0,0 +1,6 @@
+export enum TodoError {
+  LOAD = 'Unable to load todos',
```

Comment:
it will be more accurate with clarifying messages
```suggestion
   UNABLE_LOAD = 'Unable to load todos',
   UNABLE_ADD = 'Unable....',
```